articles/tech-resources/ios-universal-links-self-serve.md

```diff
@@ -0,0 +1,109 @@
+
+### Scenario 1: Universal Links Without Click Tracking
+
+Pros:
+- very simple
+Cons:
+- no click tracking on universal links
+
+This is the simplest option: if you disable SparkPost's click tracking for the universal links in your email, they work just fine with your iOS app with no further effort. This works because SparkPost does not alter untracked links in your email.
+
+#### Disabling Click Tracking
+- If you're using the SparkPost REST API, you can use the `data-msys-clicktrack` attribute to [disable click tracking for a single link in your email](https://developers.sparkpost.com/api/substitutions-reference.html#header-per-link-disabling-of-click-tracking): `<a href="http://my.universallink.example.com/path/">Open in app</a>`.
+- You can also [disable click tracking for a whole transmission request](https://developers.sparkpost.com/api/transmissions.html#header-options-attributes) using `options.click_tracking=false`.
+- [The SMTP API supports a similar flag](https://developers.sparkpost.com/api/smtp-api.html#header-open-and-click-tracking): `options.click_tracking=false`.
+- You can control [click tracking for SMTP messages](https://app.sparkpost.com/account/smtp) at the account level too.
+
+### Scenario 2: Universal Links With Click Tracking
+
+Pros:
+- minimal development effort
+Cons:
+- requires iOS app support
+
+If you'd like to use SparkPost to track when people click the universal links in your email, you can use a custom tracking domain and have your app pass each click event to SparkPost. In this scenario, you nominate a link in your email as universal by enabling click tracking on it and setting a specific "custom link sub-path". SparkPost replaces the link with a tracked version using your tracking domain and including the named path in the tracked URL. When your recipients click on the link, the domain and path match your published universal links and iOS opens your app. Your app then makes an HTTP request to the tracked URL which lets SparkPost know your recipient has visited that universal link.
+
+The setup steps are as follows:
+
+1. [Configure a custom tracking domain on your SparkPost account.](https://app.sparkpost.com/account/tracking-domains)
+1. [Use a "custom link sub-path" in your links.](https://developers.sparkpost.com/api/substitutions-reference.html#header-custom-link-sub-paths)
+1. Publish the universal links for your app using both the tracking domain and link sub-path. Your apple-app-site-association file might include the following:
+    ```
+    {
+      "applinks": {
+        "details": [
+          {
+            "appID": "<YOUR_APP_ID>",
+            "paths": [
+              "/f/open-in-app/*"
+            ]
+          }
+        ]
+      }
+    }
+    ```
```

Comment:
Also - I have no idea if we do color coding, but if we do, adding javascript after the first three backticks might make it work.